plugins/processors/awsapplicationsignals/internal/cardinalitycontrol/metrics_limiter_test.go

```diff
@@ -42,7 +41,7 @@ func TestAdmitAndRollup(t *testing.T) {
 			admittedAttributes[uniqKey.AsString()] = attr
 		} else {
 			for _, indexedAttrKey := range awsDeclaredMetricAttributes {
-				if strings.HasPrefix(indexedAttrKey, "HostedIn.") || indexedAttrKey == "Service" || indexedAttrKey == "RemoteService" {
+				if indexedAttrKey == "Environment" || indexedAttrKey == "Service" || indexedAttrKey == "RemoteService" {
```

Comment:
nit: Could use the consts defined in common similar to the `rollupMetricData` function
```suggestion
				if (indexAttrKey == common.MetricAttributeEnvironment) || (indexAttrKey == common.MetricAttributeLocalService) || (indexAttrKey == common.MetricAttributeRemoteService) {
```
or could make a function like `isSkipRollup(key string)`

Reply:
Updated to use consts